environments/dev/helm/orchestration-workflows/clinvar/values.yaml

```diff
@@ -6,8 +6,8 @@ argo:
   namespace: clinvar
   artifactBucket: broad-dsp-monster-clingen-dev-argo-archive
 chart:
-  git: true
-  ref: master
+  git: false
```

Comment:
❓ Why is `git: false` here?

Reply:
^yeah what Drew said. Just copied the args in the prod values file